tests/test_storage.py

```
@@ -946,3 +889,123 @@ async def test_get_known_branches(db_path: Path) -> None:
     assert "main" in branches["repo1"]
     assert "v2.0" in branches["repo1"]
     assert "dev" not in branches["repo1"]
+
+
+async def test_add_generation_cost(tmp_path: Path) -> None:
+    """add_generation_cost sets total_cost_usd on a variant."""
+    import docsfy.storage as storage
```

Ruff (0.15.12)

[warning] Use `from docsfy import storage` in lieu of alias

Replace with `from docsfy import storage`

(PLR0402)